README.md

```diff
@@ -171,6 +169,9 @@ go test -timeout 30s  github.com/kubernetes-csi/external-snapshotter/pkg/common-
 go test -timeout 30s  github.com/kubernetes-csi/external-snapshotter/pkg/sidecar-controller
 ```
 
+## CRDs and Client Library
+
+Volume snapshot APIs and client library are now in a separate sub-module: `github.com/kubernetes-csi/external-snapshotter/client/v3`.
```

Comment:
Is there a "go get" command they can use?

Reply:
Yes.  "go get -u github.com/kubernetes-csi/external-snapshotter/client/v3@v3.0.0". I'll add that.